public/place-image-upload.js

```diff
@@ -0,0 +1,40 @@
+(function() {
```

Comment:
@Karyum 

Do not have the same functionality in 2 files.
Please can you abstract what is happening here and only have this functionality once.

I think this comment was in the last review?

Reply:
nah it wasn't in the last review but i should have noticed that :man_facepalming: